site/tests/app/models/UserTester.php

```diff
@@ -192,7 +192,8 @@ public function testToObject() {
                 'team_invite_email' => true,
                 'team_joined_email' => true,
                 'team_member_submission_email' => true,
-                'self_notification_email' => false
+                'self_notification_email' => false,
+                'self_registration_email' => true
```

Comment:
Let's add a trailing comma here to make future changes cleaner in the diff:
```suggestion
                'self_registration_email' => true,
```

Reply:
Done